src/Files.App/Helpers/UI/AppThemeResourcesHelper.cs

```diff
@@ -27,51 +28,57 @@ public static void LoadAppResources(this IResourcesService service, IAppearanceS
 			}
 			catch
 			{
-				appearance.AppThemeBackgroundColor = "#00000000"; //migrate to new default
+				appearance.AppThemeBackgroundColor = "#00000000"; // reset to default
 				service.SetAppThemeBackgroundColor(ColorHelper.ToColor("#00000000").FromWindowsColor());
 			}
 
-			if (!string.IsNullOrWhiteSpace(appThemeAddressBarBackgroundColor) && appThemeAddressBarBackgroundColor != "#00000000")
+			if (!string.IsNullOrWhiteSpace(appThemeAddressBarBackgroundColor))
 			{
 				try
 				{
 					service.SetAppThemeAddressBarBackgroundColor(ColorHelper.ToColor(appThemeAddressBarBackgroundColor).FromWindowsColor());
 				}
 				catch
 				{
-					appearance.AppThemeAddressBarBackgroundColor = ""; //migrate to new default
+					appearance.AppThemeAddressBarBackgroundColor = ""; // reset to default
 				}
 			}
-			else
-				appearance.AppThemeAddressBarBackgroundColor = ""; //migrate to new default
```

Comment:
It's been a while since we've migrated to the new default so I think this can be removed.